builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1232,3 +1232,25 @@ def parseJsonMap(jsonString) {
 	lazyMap.putAll(jsonObj)
 	return lazyMap
 }
+
+def removePermissionAndDeleteRepo(serviceId) {
+	try {
+		def body = JsonOutput.toJson([
+			serviceId: serviceId
+		]);
+		def updatePermission = sh(script: "curl -X POST \
+				${g_base_url}/jazz/acl/policies \
+				-k -v -H \"Authorization: $auth_token\" \
+				-H \"Content-Type: application/json\" \
+				-d \'${body}\'", returnStdout: true).trim()
+		def responseJSON = parseJson(updatePermission)
+
+		if (responseJSON.data && responseJSON.data.success) {
```

Comment:
there is not going to be any data. Only success (true or false) will be returned.

Reply:
Updated